### ISSUE TITLE
Location page placement + default splash page 

### ISSUE THREAD

**Issue by panosnethood** [mazi-project/portal#113]
Hi,

I propose to create a separate Location page (see #23) both in the portal (for the users to get basic information about the MAZI Zone they are connecting) and admin pages (for editing the information displayed).

Right now some of this information is available in the Monitoring menu in both pages but I think that "Monitoring" should be separated from "Location", which is about the "identity" of the MAZI Zone.

Having this sorted, I propose to generate the splash page based on the contents of the Location page, which should also include a photo.

I think this could be an acceptable solution until more sophisticated editing of the splash page is allowed since right now is problematic that non-technical admins cannot edit the splash page at all.

**Comment by haniavis** [comment]
Yes this is nice addition.
The administrator will be able to fill in some specific fields e.g. those already existing in the monitoring tab + some more like a photo. Then this info will be used for generating the splash page in a prespecified format.
I propose to have the above solution as the permanent one, instead of allowing the admin to edit the whole splash.html file which may cause to break the html format or include insecure code.

**Comment by panosnethood** [comment]
Agreed! We could also include in the guidelines some info (e.g., the path) for experienced admins to manually change the splash page if they wish.

**Comment by mgaved** [comment]
Where is the current splash.html kept (path to the file)? thanks!

**Comment by panosnethood** [comment]
/etc/nodogsplash/htdocs/splash.html ... photos to be included in the splash page should be placed in /etc/nodogsplash/htdocs/images/. I have started a dedicated page in the guides wiki, to be updated soon (I hope): https://github.com/mazi-project/guides/wiki/Administrator-splash-page
